src/components/People/PersonCard.js

```diff
@@ -31,6 +39,9 @@ const PersonCard = ({ name, photo, surname, role, tracklist, label }) => (
               #{tracklist.toLowerCase()}
             </p>
           )}
+          {label === 'our-people' && !!about && (
+            <p className={`subtitle ${styles.description}`}>{about}</p>
```

Comment:
Maybe it could actually be a clickable mail link?

Reply:
fixed